src/definition/livechat/ILivechatContext.ts

```diff
@@ -0,0 +1,7 @@
+import { IUser } from '../users';
+import { ILivechatRoom } from './ILivechatRoom';
+
+export interface ILivechatContext {
```

Comment:
The `ILivechatContext` name sounds a little vague to me 🤔 Maybe `ILivechatEventContext` is more descriptive?